seed/python-sdk/imdb/src/seed/imdb/types/create_movie_request.py

```diff
@@ -1,14 +1,16 @@
 # This file was auto-generated by Fern from our API Definition.
 
 from ...core.pydantic_utilities import UniversalBaseModel
+import typing_extensions
+from ...core.serialization import FieldMetadata
 from ...core.pydantic_utilities import IS_PYDANTIC_V2
 import typing
 import pydantic
 
 
 class CreateMovieRequest(UniversalBaseModel):
-    title: str
-    rating: float
+    title: typing_extensions.Annotated[str, FieldMetadata(alias="title")]
+    rating: typing_extensions.Annotated[float, FieldMetadata(alias="rating")]
```

Comment:
@armandobelardo how hard is it to ignore adding an alias if the alias is the same as the normal value?

Reply:
interesting, this shouldn't be happening, will take a look

Reply:
![image](https://github.com/user-attachments/assets/92db2c7d-ced4-4e5c-994f-73e9da6ceeb7)
 🤦

Reply:
it's supposed to just be `if is_aliased`